src/PanoImageRenderer/vr/VRManager.js

```diff
@@ -0,0 +1,137 @@
+import {mat4} from "gl-matrix";
+
+const VR_DISPLAY_PRESENT_CHANGE = "vrdisplaypresentchange";
+const DEFAULT_LEFT_BOUNDS = [0, 0, 0.5, 1];
+const DEFAULT_RIGHT_BOUNDS = [0.5, 0, 0.5, 1];
+const EYES = {
+	LEFT: "left",
+	RIGHT: "right"
+};
+
+export default class VRManager {
+	get context() { return this._vrDisplay; }
+
+	constructor() {
+		this._vrDisplay = null;
+		this._frameData = new window.VRFrameData();
+
+		this._leftBounds = DEFAULT_LEFT_BOUNDS;
+		this._rightBounds = DEFAULT_RIGHT_BOUNDS;
+		this._yawOffset = 0;
+	}
+
+	destroy = () => {
+		const vrDisplay = this._vrDisplay;
+
+		this.removeEndCallback(this.destroy);
+
+		if (vrDisplay && vrDisplay.isPresenting) {
+			vrDisplay.exitPresent();
+		}
+
+		this._vrDisplay = null;
```

Comment:
@WoodNeck 

How about making a clear () method? The constructor and destroy code looks the same.

Reply:
That would be nice, at least just for now.
When we adopt typescript in this project, that would definitely bring some lint error.
But for now, I'll take it.